index.html

```diff
@@ -0,0 +1,114 @@
+
+        <label><input name="user-recommend" value="not-sure" type="radio" class="input-radio" />Not sure</label>
+      </div>
+
+      <div class="form-group">
+        <p>
+          What is your favorite feature on our platfomr?
+        </p>
+        <select id="most-like" name="mostLike" class="form-control" required>
+          <option disabled selected value>Select an option</option>
+          <option value="challenges">Challenges</option>
+          <option value="projects">Projects</option>
+          <option value="community">Community</option>
+          <option value="openSource">Open Source</option>
+          <!-- <option value="projects">Projects</option>
+          <option value="community">Community</option>
+          <option value="openSource">Open Source</option> -->
+        </select>
+      </div>
+
+      <div class="form-group">
+        <p>
+          What would you like to see improved?
+          <!-- <span class="clue">(Check all that apply)</span> -->
+        </p>
+
+        <label><input name="prefer" value="front-end-projects" type="checkbox" class="input-checkbox" />Front-end
+          Projects</label>
+        <label>
+          <input name="prefer" value="back-end-projects" type="checkbox" class="input-checkbox" />Back-end
+          Projects</label>
+        <label><input name="prefer" value="data-visualization" type="checkbox" class="input-checkbox" />Data
+          Visualization</label>
+        <label><input name="prefer" value="challenges" type="checkbox" class="input-checkbox" />Challenges</label>
+        <label><input name="prefer" value="open-source-community" type="checkbox" class="input-checkbox" />Open Source
+          Community</label>
+        <label><input name="prefer" value="gitter-help-rooms" type="checkbox" class="input-checkbox" />Gitter help
+          rooms</label>
+        <label><input name="prefer" value="videos" type="checkbox" class="input-checkbox" />Videos</label>
+        <!-- <label><input name="prefer" value="city-meetups" type="checkbox" class="input-checkbox" />City Meetups</label>
+        <label><input name="prefer" value="wiki" type="checkbox" class="input-checkbox" />Wiki</label>
+        <label><input name="prefer" value="forum" type="checkbox" class="input-checkbox" />Forum</label> -->
+        <label><input name="prefer" value="additional-courses" type="checkbox" class="input-checkbox" />Additional
+          Courses</label>
+      </div>
+
+      <div class="form-group">
+        <p>Any comments or suggestions?</p>
+        <textarea id="comments" class="input-textarea" name="comment"
+          placeholder="Enter your comment here..."></textarea>
+      </div>
+
+      <div class="form-group">
+        <button type="submit" id="submit" class="submit-button">
+          Submit
+        </button>
+        <br>
+        <br>
+        <h1 style="text-align: center;">Thank you for the information </h1>
+      </div>
+    </form>
+  </div>
+
+
+
+
```

Comment:
- [ ] kindly always check grammatical errors before submitting your comments so that your reviews look professional. In the above comments, you left some typos like "leve and reson" which you would have written "leave and reason"

please you can check out  and install Grammarly to correct grammatical errors [Grammarly](https://www.grammarly.com/grammar-check)